proto/bpfd.proto

```diff
@@ -25,9 +25,9 @@ message LoadRequestCommon {
     } ;
     string section_name = 3;
     uint32 program_type = 4;
-    optional string id = 5;
+    optional string uuid = 5;
     map<string, bytes> global_data = 6;
-    optional string map_owner_uuid = 7;
+    optional uint32 map_owner_id = 7;
```

Comment:
Agree that this is how it should be done, but we have done other API changes (proto changes) without being backwards compatible. Should we say from Release 0.3 and on we follow the programming-guides or start it mid release?

Reply:
have we made incompatible proto changes between 0.2 and 0.3?
we need to follow the guidelines at some point 😆 no strong preference on when though.

Reply:
Yes we have... I'd argue lets NOT guarentee compat until we have at least SOME stability... trying to do it now would lead to a bunch of issues... Especially with https://github.com/bpfd-dev/bpfd/pull/641 coming in right after this